### PR TITLE
Pow 9

### PR DESCRIPTION
**Pull request by derins**
# Current issues

# Code changes
1. Jira POW-9 
   general 1,2,3
   mainhome 2,3,4,6
   loads 1 - done

# Documentation changes
none

# Validation suggestions

### CODE REVIEW

File: portal/app/core/assets/js/base.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+var toggle = 1;
```
[the remark this answers is not in the document]
Comment (jongoncalves) [reply portal/app/core/assets/js/base.js:1]: to be clear, this should be a boolean, also needs a better name, like `isSidebarExpanded`

File: portal/app/core/assets/js/base.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+function toggleSideBar() {
+	$('#sidebar').toggleClass('active');
+	if (toggle === 1) {
+		toggle = 0;
+	}
+	else {
+		toggle = 1;
+	}
+	localStorage.setItem("toggle", toggle);
+}
```
Comment (jongoncalves) [inline portal/app/core/assets/js/base.js:18]:
```suggestion
$(document).ready(function() {
    let isSidebarExpanded = localStorage.getItem("isSidebarExpanded")
    if (isSidebarExpanded === null) {
        // value hasn't been set yet
        localStorage.setItem("isSidebarExpanded", "true");
    } else {
        toggleSideBar();
    }
});

function toggleSideBar() {
	$('#sidebar').toggleClass('active');
	let isSidebarExpanded = localStorage.getItem("isSidebarExpanded") === "true"
	localStorage.setItem("toggle", (!isSidebarExpanded).toString());
}
```

File: portal/app/core/assets/js/simple-diagram.js [hunk 3]
```diff
@@ -629,8 +629,16 @@
         		.attr('y', coords.y);
         }
 
+		if (opts.shape === 'hvac') {
```
Comment (jongoncalves) [inline portal/app/core/assets/js/simple-diagram.js:632]: The formatting in this looks off in GitHub. Is this an issue with your editor? What is going on?

File: portal/app/core/templates/dashboard.html [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+                    <div class="card card-body row page-header">
+                        <ul class="nav">
+                            <li class="nav-item">
+                                <a class="nav-link" href="/">
+                                    <h4 class="page-title">Resources Information</h4>
+                                </a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link icons w3-bar-item w3-button {% if resource == 'loads' %}current{% endif %}" href="/loads">
+                                    <image src="https://cdn3.iconfinder.com/data/icons/devices-and-communication-2/100/electricity_plug_female_american-512.png" height=30>
```
Comment (jongoncalves) [inline portal/app/core/templates/dashboard.html:23]: Where did you get `image` tag from? It doesn't validate and it's forced map to `img`. Please only use `img`. Also, please quote all height and width values.

File: portal/app/core/templates/partials/loads.html [hunk 5]
```diff
@@ -1,43 +1,73 @@
...
         <div class="col-sm-4 cell" align="center">
-            Kitchen: 504W
             <img src="https://static.thenounproject.com/png/81669-200.png" height="120">
         </div>
-        <div class="col-sm-4 cell" align="center">
+
+
+
+
+
```
Comment (jongoncalves) [inline portal/app/core/templates/partials/loads.html:35]: Please remove all the unnecessary whitespace

File: portal/app/core/templates/dashboard.html (hunk 4 above)
Comment (derins) [reply portal/app/core/templates/dashboard.html]: done

File: portal/app/core/templates/partials/loads.html (hunk 5 above)
Comment (derins) [reply portal/app/core/templates/partials/loads.html]: done

File: portal/app/core/assets/js/simple-diagram.js (hunk 3 above)
Comment (derins) [reply portal/app/core/assets/js/simple-diagram.js]: Fixed. I think there was inconsistency with use of indents and spaces on different lines that did not show on either of my text editors (atom and text wrangler).

**Comment by derins** [comment]
I also removed the transition property of the sidebar as it works better on page refreshes now.

File: portal/app/core/assets/js/base.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+$(document).ready(function() {
+    let isSidebarExpanded = localStorage.getItem("isSidebarExpanded");
+	// sidebar expanded by default
+    if (isSidebarExpanded === null) {
+        // value hasn't been set yet
+        localStorage.setItem("isSidebarExpanded", "true");
+    }
+	if (isSidebarExpanded === "false") {
+		// false means not expaded
```
Comment (jongoncalves) [inline portal/app/core/assets/js/base.js:9]: indentation here still looks weird...

**Comment by jongoncalves** [comment]
after the indent gets fixed, you can merge this in.

File: portal/app/core/assets/js/base.js (hunk 6 above)
Comment (derins) [reply portal/app/core/assets/js/base.js]: fixed
